src/components/@common/Button/PrimaryButton/index.tsx

```diff
@@ -1,3 +1,8 @@
+import * as Styled from './PrimaryButton.styled';
+
+export interface IButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {}
```

Comment:
```suggestion
  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
    /** 버튼 disabled */
    disabled: boolean;
    /** 버튼 width */
    width: string;
  }
```
`PrimaryButton`, `SecondaryButton` props 타입에 어떤 props를 받는지 명시하면 어떤 props를 줄 수 있는지 스타일 파일까지 들어가지 않고도 알 수 있을 것 같아 편리할 것 같습니다☺️

Reply:
전체적으로 제가 추가한 props에 대해 네이밍을 통해 명확하게 알기 어려운 것에 주석을 추가했습니다!! 😅

disabled의 경우는 제가 따로 받는게 아니라 `React.ButtonHTMLAttributes<HTMLButtonElement>`에서 제공되는 것이라 따로 주석처리하지 않았습니다..!